### DISCUSSION TITLE
connect to multiple displays in web demo using scrcpy

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#483] [Q&A]
I have a question. would it be possible in theory, to use scrcpy to connect to multiple displays at once? Ex. main display, and simulated display, side by side.

**Comment by yume-chan** [comment]
Theoretically this is possible. Multiple Scrcpy can run simultaneously, each connected to the same or different screens, as long as the device has sufficient processing power.

https://github.com/Genymobile/scrcpy/commit/4315be164823d2c8fc44b475b52af79bfee98ff1 also resolved the possible confliction when multiple Scrcpy instances are initializing at the same time.

**Reply by CorruptionVR** [reply]
can this be recreated in the web demo?

**Reply by yume-chan** [reply]
Probably won't happen.

I'm thinking about opening Scrcpy in a popup window, so it won't have those black bars when aspect ratio doesn't match. 

Maybe it's easier to allow opening multiple Scrcpy windows and let users and OS window management deal with the side-by-side part.

Opening in a new window is also the requirement of the new Document Picture-In-Picture proposal https://github.com/WICG/document-picture-in-picture.

**Reply by CorruptionVR** [reply]
actually, that is kind of what I was thinking of proposing. I was thinking of something like [airdroid](https://web.airdroid.com) or [daedalOS](https://dustinbrett.com) to create a kind of Samsung dex sort of thing. what I'm thinking is that it would use a winbox.js system, and a connect button on the taskbar with battery info, notifications and music controls
